src/System.Text.Json/tests/Serialization/ExceptionTests.cs

```diff
@@ -34,14 +34,14 @@ public static void ThrownFromSerializerFails()
         {
             try
             {
-                JsonSerializer.Parse<IDictionary<string, int>>(@"{""Key"":1, ""Key"":2}");
+                JsonSerializer.Parse<IDictionary<string, string>>(@"{""Key"":1, ""Key"":2}");
                 Assert.True(false, "Expected JsonException was not thrown.");
```

Comment:
nit: it's obvious, but please state why this should throw in this message (since we are changing the failure reason)